packages/functional-tests/pages/index.ts

```diff
@@ -61,7 +60,6 @@ export function create(page: Page, target: BaseTarget) {
     signinUnblock: new SigninUnblockPage(page, target),
     signupReact: new SignupReactPage(page, target),
     subscribe: new SubscribePage(page, target),
-    subscriptionManagement: new SubscriptionManagementPage(page, target),
```

Comment:
The underlying page is always overridden, so rather than creating the POM here, creating it in the test.